form.html

```diff
@@ -165,30 +168,33 @@ <h3 class="form__header">Your General impression:</h3>
-              <label class="form__label form__label--checkbox form__label--bolded" for="park">Slide-park</label>
-            </div>
-            <div class="form__field form__field--checkbox">
-              <input class="form__input form__input--checkbox form__input--hidden" id="mountain" name="mountain" type="checkbox" value="mountain-bell">
-              <label class="form__label form__label--checkbox form__label--bolded" for="mountain">Mountain-bell</label>
-            </div>
-            <div class="form__field form__field--checkbox">
-              <input class="form__input form__input--checkbox form__input--hidden" id="cliffs" name="cliffs" type="checkbox" value="red-cliffs">
-              <label class="form__label form__label--checkbox form__label--bolded" for="cliffs">Red cliffs</label>
+            <div class="form__item">
+              <div class="form__field form__field--checkbox">
+                <input class="form__input form__input--checkbox form__input--hidden" id="bridge" name="bridge" type="checkbox" value="devils-bridge">
+                <label class="form__label form__label--checkbox form__label--bolded" for="bridge">The Devil's Bridge</label>
+              </div>
+              <div class="form__field form__field--checkbox">
+                <input class="form__input form__input--checkbox form__input--hidden" id="park" name="park" type="checkbox" value="slide-park">
+                <label class="form__label form__label--checkbox form__label--bolded" for="park">Slide-park</label>
+              </div>
+              <div class="form__field form__field--checkbox">
+                <input class="form__input form__input--checkbox form__input--hidden" id="mountain" name="mountain" type="checkbox" value="mountain-bell">
+                <label class="form__label form__label--checkbox form__label--bolded" for="mountain">Mountain-bell</label>
+              </div>
+              <div class="form__field form__field--checkbox">
+                <input class="form__input form__input--checkbox form__input--hidden" id="cliffs" name="cliffs" type="checkbox" value="red-cliffs">
+                <label class="form__label form__label--checkbox form__label--bolded" for="cliffs">Red cliffs</label>
+              </div>
             </div>
           </div>
         </section>
 
         <section class="form__section">
           <div class="form__inner">
-            <h3 class="form__header">Upload photos</h3>
-            <div class="form__field">
-              <label class="form__label form__label--file" for="file">Choose photos</label>
+            <h3 class="form__header form__header--bottom-space">Upload photos</h3>
+            <div class="form__field form__field--bottom-space">
+              <label class="form__label form__label--file form__label--visible" for="file">Choose photos</label>
+              <label class="form__label form__label--file form__label--hidden" for="file">Drag and drop your photo here</label>
```

Comment:
Just noticed. Let's change just text without duplicating labels. For that use spans, do something along this way:
```
<label class="form__label form__label--file for="file">
<span  class="form__label-text-mobile" >Choose photos</span>
<span class="form__label-text-desktop">Drag and drop your photo here</span>
</label>
```